crates/ursa-pod/benches/e2e.rs

```diff
@@ -114,55 +118,76 @@ fn protocol_benchmarks(c: &mut Criterion) {
                 http_hyper::server_loop,
             );
         }
+
+        #[cfg(feature = "bench-quic")]
+        {
+            let mut g = c.benchmark_group(format!("QUINN UFDP/{range}"));
```

Comment:
Minor nit so both impl can be run with `QUIC`
```suggestion
            let mut g = c.benchmark_group(format!("QUINN QUIC UFDP/{range}"));
```